scripts/cronos2-devnet.yaml

```
@@ -0,0 +1,123 @@
+cronos_777-1:
+  cmd: cronosd
+  start-flags: "--trace"
+  app-config:
+    json-rpc:
+      address: "0.0.0.0:{EVMRPC_PORT}"
+      ws-address: "0.0.0.0:{EVMRPC_PORT_WS}"
+  validators:
+    - coins: 1000000000000000000stake,1000000000000000000basetcro
+      staked: 1000000000000000000stake
+      mnemonic: visit craft resemble online window solution west chuckle music diesel vital settle comic tribe project blame bulb armed flower region sausage mercy arrive release
+    - coins: 1000000000000000000stake,1000000000000000000basetcro
+      staked: 1000000000000000000stake
+      mnemonic: direct travel shrug hand twice agent sail sell jump phone velvet pilot mango charge usual multiply orient garment bleak virtual action mention panda vast
+  accounts:
+    - name: community
+      coins: 10000000000000000000000basetcro
+      mnemonic: "notable error gospel wave pair ugly measure elite toddler cost various fly make eye ketchup despair slab throw tribe swarm word fruit into inmate"
+    - name: signer1
+      coins: 20000000000000000000000basetcro
+      mnemonic: shed crumble dismiss loyal latin million oblige gesture shrug still oxygen custom remove ribbon disorder palace addict again blanket sad flock consider obey popular
+    - name: signer2
+      coins: 30000000000000000000000basetcro,3000000000000000000000011111111111111111ibc/6411AE2ADA1E73DB59DB151A8988F9B7D5E7E233D8414DB6817F8F1A01611F86
+      mnemonic: night renew tonight dinner shaft scheme domain oppose echo summer broccoli agent face guitar surface belt veteran siren poem alcohol menu custom crunch index
+    
+  
+  genesis:
+    app_state:
+      evm:
+        params:
+          evm_denom: ibc/6411AE2ADA1E73DB59DB151A8988F9B7D5E7E233D8414DB6817F8F1A01611F86
```

```suggestion
          evm_denom: basetcro
```

In current cronos logic, the `evm_denom` should be different from the ibc denom, but configure the ibc token to `cronos.params.ibc_cro_denom`, it'll be converted to the `evm_denom` automatically.

---

ok, i'll change

---

changed